changelog.txt

```diff
@@ -1,3 +1,11 @@
+Version 2.0.10
```

Comment:
Missing the following from changelog:

Content Provider changes

Reply:
This was the major change I believe in this release

Reply:
updated via https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/930/commits/9bfd0b7910976db84d091448a25f38a06bfe3bac